include/zephyr/net/wifi_nm_wpa_supp.h

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_NET_WIFI_NM_WPA_SUPP_H_
+#define ZEPHYR_INCLUDE_NET_WIFI_NM_WPA_SUPP_H_
```

Comment:
I don't think we shoudl define supplicant-specific headers in upstream Zephyr, before we actually get to have the supplicant here. IMO this should end up in NCS for now.

Reply:
Not applicable anymore since header is removed.